src/style.css

```diff
@@ -2,2 +1,79 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font for `Roboto` to ensure text displays correctly if the font fails to load.